### PR TITLE
P0 alert button

### PR DESCRIPTION
**Pull request by Omkar-Ghate**
For alert and button components. Please review the design and code.

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
good to merge...just a suggestion use variables for colors

File: pages/components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    background-color:  rgb(218, 236, 184);
+}
+.alert-danger{
+    color: var(--DANGER);
+    background-color: rgb(253, 188, 188);
+}
+.alert-warning{
+    color:var(--WARNING);
+    background-color: rgb(240, 214, 192);
+    /* background-color: var(--WARNING); */
```
Comment (archit7448) [inline pages/components/alert/alert.css:38]: just a suggestion remove commented CSS and use var

**Comment by archit7448** [comment]
looks good to merge

**Review by archit7448 — APPROVED** [review]

**Review by Muzammil-khan710 — APPROVED** [review]
A suggestion use px instead of rem where the value is 10px or less.. However it is good to merge.

**Review by sanu29 — COMMENTED** [review]
Good for me. File structure is good. Code is readable. No issues with class names and there is consistency in naming. All good to proceed.

**Pull request merged**